tensorboard/plugins/image/summary_v2.py

```diff
@@ -27,6 +27,41 @@
 def image(name, data, step=None, max_outputs=3, description=None):
     """Write an image summary.
 
+    See also `tf.summary.scalar`, `tf.summary.SummaryWriter`.
+
+    Logs a set of images to the current log directory for later analysis
+    in TensorBoard's 'Images' dashboard. Like `tf.summary.scalar` points,
+    each set of images is associated with a `step` and a `name`.  All the
+    image sets with the same `name` constitute a time series of image sets.
+
+    This example writes 2 random grayscale images:
```

Comment:
When I execute this example in eager mode against recent `tf-nightly` I get an error:

```
ValueError: .  Tensor  must have rank 4.  Received rank 3, shape (2, 8, 8)
```

I think you can resolve this by generating the images with shape `[8, 8, 1]` instead.

Not to be a drag about this, but obviously examples are only going to help users if they work :)  So let's make sure we double-check them.

Reply:
Ahh, that's embarrassing G_G.  I've double checked the latest examples.
![image](https://user-images.githubusercontent.com/2322480/111852620-e157bd00-88d4-11eb-869a-4babea0d28c6.png)

Reply:
LGTM, thanks!